src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,213 @@
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="plugin-install.php?tab=plugin-information&amp;plugin=woocommerce&amp;TB_iframe=true" class="tribe-events-admin-quick-nav__link thickbox open-plugin-details-modal"><?php esc_html_e( 'Install WooCommerce', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="plugin-install.php?tab=plugin-information&amp;plugin=the-events-calendar&amp;TB_iframe=true" class="tribe-events-admin-quick-nav__link thickbox open-plugin-details-modal"><?php esc_html_e( 'Install The Events Calendar', 'event-tickets' ); ?></a></li>
+			<?php elseif ( $has_tec ) : // ET with TEC ?>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="edit.php?page=tribe-common&tab=event-tickets&post_type=tribe_events" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Configure Settings', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="post-new.php?post_type=tribe_events" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Create RSVP', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="https://m.tri.be/1an7" target="_blank" rel="noopener noreferrer" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Setup PayPal', 'event-tickets' ); ?></a></li>
+			<?php else : // ET without TEC ?>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="edit.php?page=tribe-common&tab=event-tickets" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Configure Settings', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="plugin-install.php?tab=plugin-information&amp;plugin=the-events-calendar&amp;TB_iframe=true" class="tribe-events-admin-quick-nav__link thickbox open-plugin-details-modal"><?php esc_html_e( 'Install The Events Calendar', 'event-tickets' ); ?></a></li>
+				<li class="tribe-events-admin-quick-nav__link-item"><a href="https://m.tri.be/1an8" target="_blank" rel="noopener noreferrer" class="tribe-events-admin-quick-nav__link"><?php esc_html_e( 'Setup PayPal', 'event-tickets' ); ?></a></li>
+			<?php endif; ?>
 		</ul>
 	</div>
-</div>
 
-<div class="tribe-row">
+	<h3 class="tribe-events-admin-section-header"><?php esc_html_e( 'Helpful Resources', 'event-tickets' ); ?></h3>
 
-	<div class="tribe-half-column">
-		<h4><?php esc_html_e( 'The Latest and Greatest', 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'Frequent maintenance releases keep your ticket sales running smoothly.', 'event-tickets' ); ?> <a href="http://m.tri.be/1a9z" target="_blank"><?php esc_html_e( 'View the latest changelog', 'event-tickets' ); ?></a>.</p>
-		<p><?php esc_html_e( 'Gearing up with Gutenberg?', 'event-tickets' ); ?> <a href="http://m.tri.be/1aa0" target="_blank"><?php esc_html_e( 'Get the latest block editor news', 'event-tickets' ); ?></a>.</p>
+	<?php /* Video was not yet ready
```

Comment:
* :no_entry_sign: Opening PHP tag must be on a line by itself
* :no_entry_sign: Block comment text must start on a new line

<sub>hash: 5a1b7e6b36f4f788e1af77aec368a22e</sub>

Reply:
```suggestion
	<?php
	/* Video was not yet ready
```